jsapp/js/searches.es6

```diff
@@ -80,7 +80,6 @@ function SearchContext(opts={}) {
     },
     onDeleteAssetCompleted (asset) {
       var filterOutDeletedAsset = ({listName}) => {
-        // TODO: look into why sometimes this.state[listName] is not defined
         if (this.state[listName] != undefined) {
```

Comment:
Do you think removing this `// TODO` should leave an explanation comment?

Reply:
No need. FYI, the search should get an overhaul soon-ish.

Reply:
Ok